Lib/shutil.py

```diff
@@ -231,16 +231,22 @@ def _stat(fn):
 def _islink(fn):
     return fn.is_symlink() if isinstance(fn, os.DirEntry) else os.path.islink(fn)
 
-def copyfile(src, dst, *, follow_symlinks=True):
+def copyfile(src, dst, *, follow_symlinks=True, same_file=False):
```

Comment:
in general looks fine, I would propose more explicit name eg `ignore_same_file`

Reply:
You are right! I have change the name of argument to `ignore_same_file`!